test/e2e/utils/config.go

```diff
@@ -5,11 +5,11 @@ package utils
 import "time"
 
 const (
-	// syncTimeout is the maximum time the sync loop can last. In normal circumstances this is in the order
+	// SyncTimeout is the maximum time the sync loop can last. In normal circumstances this is in the order
 	// of seconds but there are special cases we need to account for. Like when the sync loop generates new keys
 	// and signs certificates it can take several seconds. When the CI creates multiple clusters in parallel on
 	// a constrained CPU, one cert can easily take over 30s.
-	syncTimeout        = 2 * time.Minute
+	SyncTimeout        = 2 * time.Minute
```

Comment:
stale commit squash from different PR

Reply:
I think that was just the commits from the previous PR, it's intentionally exposed